dockerfiles/puppeteer-context/mina_daemon_puppeteer.py

```diff
@@ -94,12 +94,13 @@ def start_daemon():
   log("touching /root/daemon-active" )
   Path('daemon-active').touch()
   log("daemon fully started" )
-  # log_event("node_initialized", "node has been initialized")
+  log_event("node_started", "node has been started")
```

Comment:
Why this renaming in this PR?

Reply:
As I said in the PR description, this isn't a renaming per se, I split out node_initialized into node_initialized and node_started to differentiate between actuality and intention.